app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,27 @@
  *= require_tree .
  *= require_self
  */
+
+ footer {
+   position: absolute;
+   bottom: 0;
+   left: 0;
+   width: 100%;
+   font-family: "futura-pt", Futura, "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
+   font-weight: 500;
+   letter-spacing: 0.05em;
+   text-rendering: optimizeLegibility;
```

Comment:
Some of this styling seems like it should be global, rather than specific to the footer. Did you come up with it yourself or did it come from somewhere?